openvdb_points/tools/AttributeArray.h

```diff
@@ -66,6 +66,8 @@ namespace OPENVDB_VERSION_NAME {
 // Add new typedef for a Name pair
 typedef std::pair<Name, Name> NamePair;
 
+typedef uint32_t StringType;
```

Comment:
I find it a little confusing to be defining a type called StringType that isn't actually a string - perhaps rename to StringIndexType or StringKeyType?